tools/cmgen/src/cmgen.cpp

```diff
@@ -799,8 +802,13 @@ void iblRoughnessPrefilter(const utils::Path& iname,
         }
     }
 
-    if (g_ktxContainer) {
+    if (g_ktx_output) {
         using namespace std;
+        if (g_coefficients) {
+            ostringstream sstr;
+            outputSh(sstr, g_coefficients, g_sh_compute);
+            container.setMetadata("sh", sstr.str().c_str());
```

Comment:
Should we save the SH as just floats in the metadata so it's easier to parse on the other side?

Reply:
Good point, added a new commit that simplifies this.